aiida/tools/dbexporters/tcod.py

```diff
@@ -1030,6 +1030,8 @@ def export_cifnode(what, parameters=None, trajectory_index=None,
         function_args['node'] = node
     if parameters is not None:
         function_args['parameters'] = parameters
+    else:
+        function_args['parameters'] = ParameterData(dict={})
```

Comment:
Just to understand, will this be ever triggered? (My guess: only if it's not working as an Inline and the user passes None, while the user can never pass None if it's an inline?)

Reply:
Yes that is also my understanding: it will only be hit if `None` is passed in explicitly, which can occur if non inline functions have that as a default and call this function while just passing through their arguments. Shall I merge this and open a separate issue for a discussion about allowing `None` as an input for `workfunctions` and therewith `InlineCalculations`?

Reply:
Yes, thanks!